release/release_workflow.wdl

```diff
@@ -40,7 +40,7 @@ task do_release {
        # Generate new scaladoc
        sbt 'set scalacOptions in (Compile, doc) := List("-skip-packages", "better")' doc
        git checkout gh-pages
-       for subproj in cwl wdl wom; do
+       for subproj in cwl wom; do
```

Comment:
Not sure I would fix it this way, we will have scaladocs for WDL once WDL and WOM are separated.  I would make sure the directory exists instead.